src/components/browse/datasetlist/datasetheader/datasetdetails/datasetDetails.tsx

```diff
@@ -14,21 +14,30 @@ const DatasetDetails = (props: dataSetDetailsProps) => {
   let hit = props.hit;
 
   return (
-    <div className="fs-8">
+    <div className="fs-9">
       <Row>
-        <Col lg md sm xl xs xxl="10">
+        <Row className="pe-0">
+          <Col>
           <p className="my-0">
             <span className="fw-bold">Dataset ID:&nbsp;</span>
-            {hit.content.accession}
-
-            <br />
-            <span className="fw-bold">Description:&nbsp;</span>
-            {hit.content.description}
+            <span style={{userSelect: "all"}}>{hit.content.accession}</span>
+          </p>
+          <p>
+            <span className="fw-bold">Full title:&nbsp;</span>
```

Comment:
This field is not part of the wireframes. Please remove this.

Reply:
I have made this change because I made the title is trimmed so as to not have multi-line titles in the header, therefore, it was necessary to have the full title inside the details area. If the wireframe team has an issue with this change, I can address them.

Reply:
Can you check with Koray/Hazal/Leon about this field? I would like to merge this PR soon.

Reply:
Looks redundant, lets not have this field for now and bring it to the UI-UX team tomorrow during the meeting. For now remove the field.

Reply:
This should have been fixed as per discussion.